merge/choice.py

```diff
@@ -2,10 +2,10 @@
 
 
 class Choice(Enum):
-    undesided = 0
+    undecided = 0
```

Comment:
No, a resolved conflict is ether `left`, `right` or `both` (see the `is_resolved` method)

`Choice.undecided` is the opposite of  a resolved conflict